src/components/policy/src/policy/policy_table/table_struct/types.cc

```diff
@@ -489,6 +489,24 @@ ModuleConfig::ModuleConfig(const Json::Value* value__)
     vehicle_model(impl::ValueMember(value__, "vehicle_model")),
     vehicle_year(impl::ValueMember(value__, "vehicle_year")) {
 }
+
+void ModuleConfig::SafeCopyFrom(const ModuleConfig& from) {
```

Comment:
As for me this is more obvious way.
Because it would be really not evident if the `operator=` would avoid the copy of some fileds.
That's why I ve implemented this method for PASA code instead of overloading `operator=`